src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unnecessaryparentheses/InputUnnecessaryParenthesesLambdas.java

```diff
@@ -15,7 +15,7 @@
 public class InputUnnecessaryParenthesesLambdas {
     int foo(int y) {
         MathOperation case1 = (x) -> x + x; // violation 'Unnecessary paren.* around lambda value'
-        MathOperation case2 = (x) -> { return x + x; }; // violation 'paren.* around lambda value'
+        MathOperation case2 = (x) -> { return ((x + x)); }; // violation 'paren.* around lambda.*'
```

Comment:
Please do not modify existing test cases, except to change violation message.

Reply:
Fixed

Reply:
Resolved.